<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1410</width>
    <height>1038</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Лабораторная работа №1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QLabel" name="setAddLabel">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Множество:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="xAddLineEdit"/>
    </item>
    <item row="5" column="1" colspan="3">
     <widget class="QPushButton" name="cleanButton">
      <property name="text">
       <string>Очистить</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+D</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5" colspan="4">
     <widget class="QPushButton" name="cancleButton">
      <property name="text">
       <string>Отмена</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+Z</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="xAddLabel">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>X:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="pointNumLabel">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>№ точки:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5" colspan="4">
     <widget class="QPushButton" name="solveButton">
      <property name="styleSheet">
       <string notr="true">color = rgb(143, 100, 164)</string>
      </property>
      <property name="text">
       <string>Решить</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+S</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="yAddLineEdit"/>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="setAddLineEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="yAddLabel">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Y:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLineEdit" name="pointDelLineEdit"/>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QPushButton" name="addButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+V</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5" colspan="3">
     <widget class="QPushButton" name="delButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
      <property name="shortcut">
       <string>Ctrl+V</string>
      </property>
     </widget>
    </item>
    <item row="4" column="7" alignment="Qt::AlignRight">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item row="4" column="1" colspan="3">
     <widget class="QLabel" name="labelCanvas">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">bg: rgb(255, 255, 255)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1410</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="infoMenu">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="progInfo"/>
    <addaction name="authorInfo"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <addaction name="infoMenu"/>
  </widget>
  <action name="progInfo">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="authorInfo">
   <property name="text">
    <string>Об авторе</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
